src/main/java/uk/ac/ebi/subs/validator/data/SingleValidationResult.java

```diff
@@ -1,46 +1,28 @@
 package uk.ac.ebi.subs.validator.data;
 
 import lombok.ToString;
+import uk.ac.ebi.subs.validator.data.structures.SingleValidationResultStatus;
+import uk.ac.ebi.subs.validator.data.structures.ValidationAuthor;
 
 /**
   * Validation entity result document to store an entity validation result for a specific rule set
   */
-@ToString(callSuper = true)
-public class SingleValidationResult extends AbstractValidationResult implements Identifiable {
-
-    private int version;
-    private String uuid;
-    private String validationResultUUID;
+@ToString
+public class SingleValidationResult {
 
     private ValidationAuthor validationAuthor;
 
-    public SingleValidationResult() {
-    }
+    private SingleValidationResultStatus validationStatus = SingleValidationResultStatus.Pending;
 
-    public SingleValidationResult(ValidationAuthor validationAuthor, String entityUuid) {
-        this.validationAuthor = validationAuthor;
-        this.setEntityUuid(entityUuid);
-        this.setValidationStatus(ValidationStatus.Pending);
-    }
+    private String message;
```

Comment:
I can't see any usage of this private field

Reply:
It's used by the validators to set the error message